workers/unity/Packages/io.improbable.gdk.debug/.codegen/Source/Generators/DebugExtensions/FieldTypeHandler.cs

```diff
@@ -136,6 +136,22 @@ public string ToUiFieldUpdate(UnityFieldDetails fieldDetails, string fieldParent
             }
         }
 
+        public string ToSetCollectionVisibility(UnityFieldDetails fieldDetails, string fieldParent)
+        {
+            var uiElementName = $"{fieldDetails.CamelCaseName}Field";
+            switch (fieldDetails.FieldType)
+            {
+                case SingularFieldType singularFieldType:
+                    return string.Empty;
+                case OptionFieldType optionFieldType:
+                case ListFieldType listFieldType:
+                case MapFieldType mapFieldType:
+                    return $"SetVisibility({fieldParent}.{fieldDetails.PascalCaseName}, {uiElementName});";
```

Comment:
Rather than calling these every update, can they instead be reactive? I.e. - only change when the value is actually toggled.

Reply:
Fixed it, its now called only when the hide option is changed and when the user selects an entity in the worker inspector.